codice/ddf
# Removes Hazelcast from standard framework

#### What does this PR do?
Removes the Hazelcast dependency from catalog-core-standardframework. 

#### Who is reviewing it? 
@lavoywj 
@abel-connexta 

#### Select relevant component teams: 
@codice/core-apis 
@codice/io 


#### Ask 2 committers to review/merge the PR and tag them here.
@pklinef 
@shaundmorris 

#### How should this be tested?
- Ingest an image file by sending a multipart `POST` request to `/services/catalog` (see https://connexta.slack.com/archives/C01NZCY4L0G/p1619223771025400 for details how to do that).
- Discover the resource using the simple search UI (https://localhost:8993/search/simple/#).
- Ensure successful downloads of the record **as a resource** are possible. The browser should open the original image in a new tab. 

#### Any background context you want to provide?
The Hazelcast resource cache isn't being used so let's simplify things and get rid of it. 

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
